google-finance/src/components/Common/Footer.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+import logoImage from '../../assets/img/image_logo.svg';
+import { FooterLogoImage, LineImage } from '../../assets/index';
+
+const Footer = () => {
+  return (
+    <FooterContainer>
+      <FooterHeader>
+        <img src={FooterLogoImage} alt="구글금융 메뉴" />
+        <span>도움말</span>
+        <span>의견 보내기</span>
+        <span>개인정보처리방침</span>
+      </FooterHeader>
+      <FooterSub>
+        <FooterText>google.com</FooterText>
+        <img src={LineImage} alt="글자 사이 라인" />
+        <FooterText>개인정보 보호</FooterText>
+      </FooterSub>
+    </FooterContainer>
+  );
+};
+
+const FooterContainer = styled.nav`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  position: absolute;
+
+  width: 99%;
+  height: 11.5rem;
+  padding: 0 1.5rem;
+  bottom: 0;
+
+  font-family: ${(props) => props.theme.fonts.roboto_12};
```

Comment:
혹시 font-family: ~~ 이렇게 했을때 roboto_12 폰트 크기랑 다 잘 적용됐어??
난 어제 이렇게했을땐 제대로 적용안되길래
```suggestion
  ${(props) => props.theme.fonts.roboto_12};
```
그냥 일케만 하니까 적용되더라구!

Reply:
똑같은 것 같은데 통일성을 위해서 폰트패밀리 빼도록 할게!! 고마우잉!!